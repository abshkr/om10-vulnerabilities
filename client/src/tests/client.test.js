import React from "react";
import renderer from "react-test-renderer";

import {
  Breadcrumbs,
  Calendar,
  Container,
  Download,
  Drawer,
  Filter,
  Footer,
  FormModal,
  IButton,
  Loading,
  Locked,
  Navigation,
  Page,
  DataTable,
  Title
} from "../components";

test("Breadcrumbs - Init", () => {
  const component = renderer.create(<Breadcrumbs />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
