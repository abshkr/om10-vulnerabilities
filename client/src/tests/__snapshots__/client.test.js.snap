// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs - Init 1`] = `
<div
  className="breadcrumbs"
>
  <div
    className="ant-page-header"
  >
    <div
      className="ant-breadcrumb"
    >
      <span>
        <span
          className="ant-breadcrumb-link"
        >
          <a
            href="#/"
          >
            OMEGA 5000
          </a>
        </span>
        <span
          className="ant-breadcrumb-separator"
        >
          /
        </span>
      </span>
      <span>
        <span
          className="ant-breadcrumb-link"
        >
          <a
            href="#/first"
          />
        </span>
        <span
          className="ant-breadcrumb-separator"
        >
          /
        </span>
      </span>
      <span>
        <span
          className="ant-breadcrumb-link"
        >
          <span />
        </span>
        <span
          className="ant-breadcrumb-separator"
        >
          /
        </span>
      </span>
    </div>
  </div>
</div>
`;
